### DISCUSSION TITLE
Need for Task status listener

### DISCUSSION THREAD

**Discussion by charybr** [Netflix/conductor#3587] [Ideas]
When the status of a task changes, we need a mechanism to notify. Just like we have WorkflowStatusListener interface, we can have TaskStatusListener interface.
Is this feature already there ?

**Comment by v1r3n** [comment]
Hi @charybr this feature does not exist in the open source yet, but we have it available in the enterprise version.  We plan to push to OSS in the near future.

**Reply by charybr** [reply]
thank you Viren! that will help.

**Comment by charybr** [comment]
Same question was asked in the past as well
https://github.com/Netflix/conductor/discussions/2322
